msteams-platform/apps-in-teams-meetings/build-extensible-conversation-for-meeting-chat.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Build extensible conversation for meeting chat
+author: v-sdhakshina
+description: In this article, learn how to build extensible conversation for Microsoft Teams meeting chat with bots, cards and message extensions.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+---
+
+# Build extensible conversation for meeting chat
```

Comment:
Im a bit confused about the purpose of this page, as it seems duplicative of general documentation about bots and messaging extensions. The information specific to meetings is hidden in the paragraph below:
> Bots that are enabled in group chat scope start functioning in meetings.

Reply:
the page is intended to cover Bot, ME, AC and Task Module related functionality reference, Added a table with feature accessibility by user role for better clarification.